test/paper-toolbar-v1-to-v2/child-non-whitespace-text_before.html

```diff
@@ -0,0 +1,3 @@
+<paper-toolbar>
+  This text node will be wrapped in a &lt;span slot="top"&gt;.
```

Comment:
I'd like to see a test case with multiple children, a mixture of text nodes and elements, and some text nodes with newlines in their main content

Reply:
Added.